styleguide/labels-projects.md

```diff
@@ -0,0 +1,119 @@
+
+![:books: Area on dark blue background](./images/area.png "Prefix for .NET Guide area labels")
+
+Each .NET Guide is noted with the `:books: Area` prefix and has a dark blue background. Here are current issues filtered for each of the .NET guides.
+
+- [API Reference](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20API%20Reference)
+- [Azure .NET SDK](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20Azure%20.NET%20SDk)
+- [C# Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20C%23%20Guide)
+- [Desktop Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20Desktop%20Guide)
+- [F# Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20F%23%20Guide)
+- [ML.NET Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20ML.NET%20Guide)
+- [.NET Architecture Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20.NET%20Architecture%20Guide) - Could be removed
+- [.NET Core Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20.NET%20Core%20Guide)
+- [.NET for Apache Spark Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20.NET%20for%20Apache%20Spark%20Guide)
+- [.NET Framework Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20.NET%20Framework%20Guide)
+- [.NET Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20.NET%20Guide)
+- [Roslyn API Reference](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20Roslyn%20API%20Reference) - could be removed.
+- [Visual Basic Guide](https://github.com/dotnet/docs/labels/%3Abooks%3A%20Area%20-%20Visual%20Basic%20Guide)
+
+#### Search one section of a guide
+
+![:card_file_box: Area on medium blue background](./images/technology.png "Prefix for .NET Guide sub-area labels")
+
+The .NET guides are large, so these labels further limit the scope by a section of a guide. Each .NET Guide subarea is noted with the `:card_file_box: Technology` prefix and has a medium blue background. Many of these labels apply to multiple guides. Add one of these labels to your filter after scoping your search to one of the areas described in the previous section.
+
+- AppCompat
+- Async Task
+- C# Advanced concepts
+- C# compiler
+- C# Fundamentals
+- C# Get Started
+- C# Language Reference
+- C# Null safety
+- C# What's new
+- CLI
+- Data Access
+- Docker
+- Installers
+- LINQ
+- NCL
+- .NET Standard
+- Roslyn APIs
+- Security
+- WCF
+- WF
+- WIF
+- WinForms
+- WPF
+
+### Releases
+
+![:checkered_flag: Release: on dark yellow](./images/release.png "Prefix for release labels")
+
+Issues tagged for a specific release are noted with the `:checkered_flag: Release:` prefix and have a dark yellow background. 
+
+- .NET Core 2.1
+- .NET Core 2.2
+- .NET Core 3.0
+- .NET Framework 4.7.2
```

Comment:
I thought we'd remove the Framework ones and 2.1 now

Reply:
I haven't removed them yet, but that's the plan. I'll remove them from this article.